YellowNotes.Core/Utility/GetNotesConfig.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+
+namespace YellowNotes.Core.Utility
+{
+    public class GetNotesConfig
```

Comment:
name of class is quite accurate, I wouldn't say it's a config

Reply:
How would you name it then? 😄

Reply:
it comes from api, so it's a dto. it could be `NoteSearchRequestDto` or `NoteQueryDto`, more sth like that.
would be nice to have abstract class `PagingRequest` and inherit from it